src/CommunityToolkit.Maui/Views/RatingView/RatingView.Shared.cs

```diff
@@ -0,0 +1,380 @@
+﻿using System.Windows.Input;
+using CommunityToolkit.Maui.Core.Primitives;
+using CommunityToolkit.Maui.Core.Primitives.Defaults;
+using Microsoft.Maui.Controls.Shapes;
+
+namespace CommunityToolkit.Maui.Views.RatingView;
+
+/// <summary>RatingView control.</summary>
+public class RatingView : Border
+{
+	#region Private Properties
```

Comment:
we don't use regions, please remove them

Reply:
Alight